src/ncp/ncp_openthread.cpp

```diff
@@ -285,6 +286,9 @@ otError ControllerOpenThread::ApplyFeatureFlagList(const FeatureFlagList &aFeatu
 #if OTBR_ENABLE_DHCP6_PD
     otBorderRoutingDhcp6PdSetEnabled(mInstance, aFeatureFlagList.enable_dhcp6_pd());
 #endif
+#if OTBR_ENABLE_LINK_METRICS_TELEMETRY
+    otLinkMetricsManagerSetEnabled(mInstance, aFeatureFlagList.enable_link_metrics_mgr());
```

Comment:
I wonder where is the other place that the Link Metrics Manager is set to enabled?

Reply:
There's a configuration option `OPENTHREAD_CONFIG_LINK_METRICS_MANAGER_ON_BY_DEFAULT` to decide if it's turned on/off by default: https://github.com/openthread/openthread/blob/main/src/core/utils/link_metrics_manager.cpp#L58